rustainers/src/runner/inner.rs

```diff
@@ -478,7 +613,10 @@ impl<'a> CreateAndStartOption<'a> {
         let ports = &image.port_mappings;
         let remove = option.remove;
         let name = option.name();
-        let network = &option.network;
+        let network = option
+            .network
+            .as_ref()
+            .map_or_else(|| Cow::Owned(Network::default()), Cow::Borrowed);
```

Comment:
Maybe replace the network field into option by an accessor. instead of having is_none in several places in the code.

Reply:
I don't understand what you have in mind.